Core/Base/ConsumerModule.py

```diff
@@ -0,0 +1,43 @@
+"""
+  DIRAC class which is the abstract class for the consumers
+
+  All DIRAC consumers must inherit from the basic class ConsumerModule
+  and override its methods
+"""
+
+from DIRAC import S_ERROR
+#from DIRAC.Core.Utilities.MQConnector import MQConnector
+
+class ConsumerModule(object):
+  """ Base class for all consumer modules
+
+      This class is used by the ConsumertReactor class to steer the execution of
+      DIRAC consumers.
+  """
+
+  def __init__( self ):
```

Comment:
I believe __init__() should have the following signature:
```
def __init__( self, consumerName, loadName, baseConsumerName = False, properties = {} ):
```
And these info should be used for several thing, e.g. for the subLogger (that should be in self.log), and for defining a self.__configDefaults dictionary (same as for AgentModule). This dictionary should be filled by the CS parameters defined for the consumer, and this includes e.g. the MQ host, and the queue name (these last parameters should be passed to RabbitMQConnector).